cmd/experimental/kjobctl/apis/v1alpha1/volumebundle_types.go

```diff
@@ -17,11 +17,35 @@ limitations under the License.
 package v1alpha1
 
 import (
+	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+type MountPoint struct {
+	// Inlines Container’s volume mounts.
+	corev1.VolumeMount `json:",inline"`
+
+	// Environment variables pointing to locations on the mount.
+	// These values will be copied to pods/containers environment variables.
+	// +optional
+	Env []string `json:"env,omitempty"`
```

Comment:
@trasc Shouldn't it be just a one env? Or I something missed?

Reply:
Also if it's array maybe name it as `Envs`?

Reply:
@mwielgus WDYT?

Reply:
It is supposed to be an array as there might be multiple important locations on the volume. Please make it Envs.

Reply:
Renamed.

Reply:
Thanks

Reply:
An environment it's a collection of environment variables. :), I think the intent is to have multiple env-variables that will be placed in a single environment.